dhis_android_analytics/src/main/java/dhis2/org/analytics/charts/providers/ChartCoordinatesProviderImpl.kt

```diff
@@ -204,7 +205,7 @@ class ChartCoordinatesProviderImpl(
                 }
 
                 GraphPoint(
-                    eventDate = Date(),
+                    eventDate = GregorianCalendar(2021, 0, 1).time,
```

Comment:
Would this create any problem in the future by setting that specific date?

Reply:
Nope. The date assigned here is not used (the GraphPoint uses the position param) but can't be null. Assigning Date() caused the adapter to think that items always changed (newItem == oldItem always returned false as Date() provided two different timestamps). Setting a fixed date fixes this.
In the future, we would need to refactor the class to check if we can remove the eventDate param in GraphPoint.